test/io.jl

```diff
@@ -44,17 +41,25 @@ module TestIO
                    B = 'a':'c',
                    C = ["A", "B", "C"],
                    D = CategoricalArray('a':'c'),
-                   E = NullableCategoricalArray(["A", "B", "C"]),
-                   E = NullableArray(1:3),
-                   F = NullableArray(fill(Nullable(), 3)),
-                   G = fill(Nullable(), 3))
+                   E = CategoricalArray(["A", "B", null]),
+                   F = Vector{Union{Int, Null}}(1:3),
+                   G = nulls(3),
+                   H = fill(null, 3))
 
-    answer = Sys.WORD_SIZE == 64 ? 0x3bd4a4a099ae8aac : 0x5ecdc4bb
-    @test hash(sprint(printtable, dt)) == answer
+    DRT = CategoricalArrays.DefaultRefType
+    # FIXME CategoricalValue printing differences between 0.6 and 0.7
```

Comment:
If you uncomment this, wouldn't tests pass on 0.6? That would be OK for now I'd say, anyway 0.7 is a moving target.